### ISSUE TITLE
idea

### ISSUE THREAD

**Issue by chmich** [nilsteampassnet/TeamPass#2710]
Dear Nils,

often, i have a customer, and the customer just want to tell me a password, but it is only once, and i dont want to give a TeamPass-Login, just for this case.

Then, he's doing that on SMS, Email, telephone, or whatever.

would that feature be a idea:?

In The app, there could be a page, secured just by captcha (no login required), and a message-field, where the user can post. Then its transmitted secure (https). When i log in TeamPass, i can see the new posts?
.. and  nice-to-have: In case, a Notification could be desplayed on my screen

Thanks, 
Christian

 
### Steps to reproduce
1.
2.
3.

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

### Server configuration
**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**Teampass version:** 

**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**
PLEASE attach to this issue the file `/includes/config/tp.config.php`.

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
```
Insert your webserver log here
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by ozawa-hi** [comment]
Have you seen the following statement when creating a new issue here? You've posted your feature request to a wrong place.

> If you have any FEATURE REQUEST or question, please use https://teampass.userecho.com/

**Comment by chmich** [comment]
thanks ozawa, now its here: https://teampass.userecho.com/communities/1/topics/201-easy-and-secure-messages
